### PR TITLE
README template

### PR DESCRIPTION
**Pull request by dannywillems**
- add NVM solution
- remove useless comment (through USB)
- add usage information for Makefile.local

### CODE REVIEW

File: template.distillery/README.md [hunk 1]
```diff
@@ -110,13 +110,18 @@ Build the mobile applications
 
 ### For all mobile platforms:
 
-- Install npm (on Debian and Ubuntu, `sudo apt-get install npm`).
+- Install npm (on Debian and Ubuntu, `sudo apt-get install npm`). If you have
+  some issues with your package manager, you can also use
```
Comment (vasilisp) [inline template.distillery/README.md]: _you have issues_ (delete _some_)

File: template.distillery/README.md [hunk 2]
```diff
@@ -183,7 +188,8 @@ be created by getting all the necessary files (js, etc) from a server.
 This may be used to create a mobile app for an which has not been
 compiled locally. With `APP_REMOTE=no`, the local files will be used.
 
-Replace ${YOUR_SERVER} by ${YOUR_IP_ADDRESS}:8080 in the following
+The pointed server address is given in the variable `APP_SERVER`.
```
Comment (vasilisp) [inline template.distillery/README.md]: _Pointed_ sounds wrong. _Remote_ server address maybe?

File: template.distillery/README.md [hunk 3]
```diff
@@ -222,3 +228,24 @@ you update the server:
 ```
 make APP_SERVER=http://${YOUR_SERVER} APP_REMOTE={yes|no} chcp
 ```
+
+## Use Makefile.local file.
+
+You need to define `APP_REMOTE` and `APP_SERVER` each time you want to build
+mobile applications or to update it.
```
Comment (vasilisp) [inline template.distillery/README.md]: _... the mobile application, or to update it_

File: template.distillery/README.md [hunk 4]
```diff
@@ -222,3 +228,24 @@ you update the server:
...
+
+If you don't want to pass every time the variables `APP_SERVER` and
+`APP_REMOTE`, you can change the values of these variables in
```
Comment (vasilisp) [inline template.distillery/README.md]: _pass the variables `APP_SERVER` and `APP_REMOTE` every time_ (move _every time_)

File: template.distillery/README.md [hunk 5]
```diff
@@ -222,3 +228,24 @@ you update the server:
...
+`Makefile.local.example` and rename this file in `Makefile.local`. With this
```
Comment (vasilisp) [inline template.distillery/README.md]: _rename this file to_

File: template.distillery/README.md [hunk 6]
```diff
@@ -222,3 +228,24 @@ you update the server:
...
+method, `APP_REMOTE` and `APP_SERVER` variables are not mandatory when you build
```
Comment (vasilisp) [inline template.distillery/README.md]: * The variables `APP_REMOTE` and `APP_SERVER` are not mandatory
* _With this method, ..._ -> _This way, ..._ (simpler)

File: template.distillery/README.md [hunk 7]
```diff
@@ -222,3 +228,24 @@ you update the server:
...
+or update the mobile application. You can use:
+```
+make chcp
+make run-android
+make run-ios
+...
+```
+
+It is recommended to use this file to add rules or variables only used for
+local development and it must not be deployed or shared (by default, this file
```
Comment (vasilisp) [inline template.distillery/README.md]: I would rephrase this, _e.g._, as follows: _this file is meant for rules and variables that are only relevant for local development..._

**Comment by dannywillems** [comment]
@vasilisp All fixed.
